src/ResultsActivity/ResultsActivityContainer.js

```diff
@@ -27,31 +30,24 @@ class ResultsActivityContainer extends Component {
     this.handlePrint = this.handlePrint.bind(this)
     this.setColumnSort = this.setColumnSort.bind(this)
     this.getActivityList = this.getActivityList.bind(this)
-    this.update = this.update.bind(this)
+    this.onListCriteriaChange = this.onListCriteriaChange.bind(this)
+    this.reload = this.reload.bind(this)
     this.state = {
       redactedPrint: false,
     }
   }
 
   async componentDidMount() {
```

Comment:
This never really worked and resulted in the data that drives the page to be fetched 2-3 times on each state change.